plugins/modules/pacemaker_cluster.py

```diff
@@ -26,169 +29,133 @@
   state:
     description:
       - Indicate desired state of the cluster.
-    choices: [cleanup, offline, online, restart]
+    choices: [offline, online, restart, maintenance]
     type: str
-  node:
+    required: true
```

Comment:
This is also a breaking change. As odd as it seems, someone may have a playbook with `{{ state: var | default(omit) }}` in which case they effectively "disable" the task. I cannot say I like that design choice, but if this is changed without a deprecation period, it can break existing playbooks. Please deprecate it as well.